keygen/src/keygen.rs

```diff
@@ -41,6 +41,86 @@ struct GrindMatch {
     count: AtomicU64,
 }
 
+const WORD_COUNT_ARG: ArgConstant<'static> = ArgConstant {
+    long: "word-count",
+    name: "word_count",
+    help: "Specify the number of words that will be present in the generated seed phrase",
+};
+
+const LANGUAGE_ARG: ArgConstant<'static> = ArgConstant {
+    long: "language",
+    name: "language",
+    help: "Specify the mnemonic lanaguage that will be present in the generated seed phrase",
+};
+
+const NO_PASSPHRASE_ARG: ArgConstant<'static> = ArgConstant {
+    long: "no-bip39-passphrase",
+    name: "no-passphrase",
```

Comment:
```suggestion
    name: "no_passphrase",
```

Reply:
Thank you and your eagle eye!

Reply:
I feel embarrassed by the number of typo and silly mistakes in this change, but to be honest I found it a little hard to keep track of the numerous strings and their subtle differences.  I think probably I always would have used dashes or underscores, not mixed and matched, precisely because I get so easily confused ...

Reply:
Also I was not exhaustive in my testing of the command line options.  I thought I covered the stuff that I had changed, but obviously typos mean that I changed things I thought I hadn't and my testing was not comprehensive enough.  Mea culpa.

Reply:
Not to worry, that's what the CI test suite is for!

> I found it a little hard to keep track of the numerous strings and their subtle differences. I think probably I always would have used dashes or underscores, not mixed and matched, precisely because I get so easily confused ...

Oh totally, it's tricky to keep track of the conventions, particularly inside strings where rustfmt can't do the work! We do try to stay consistent -- kebab case for cli params, otherwise snake case to match rust variables -- but there may be existing mistakes in the code base 🤕